docs/spec/build.gradle

```
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
```

Shall we check the other places as well?